ui/app/AppMain.qml

```diff
@@ -339,8 +341,11 @@ RowLayout {
             property var _web3Provider: web3Provider
         }
 
-        TimelineLayout {
+        Loader {
             id: timelineLayoutContainer
+            sourceComponent: TimelineLayout {}
```

Comment:
I just want to be sure, we don't need to put a surrounding `Component {}`? If so, TIL

Reply:
Yea, I was surprised about it too! Seems to work